.gitignore

```diff
@@ -36,15 +33,19 @@ node_g
 icu_config.gypi
 .eslintcache
 node_trace.*.log
-coverage/
-
+/coverage/
 /out
 
 # various stuff that VC++ produces/uses
 Debug/
-!**/node_modules/debug/
-!deps/v8/src/debug/
+!/deps/v8/test/debugger/debug/
+!/deps/v8/src/base/debug
+!/deps/v8/src/debug
+/deps/v8/src/debug/obj
+deps/v8/src/inspector/Debug/
```

Comment:
if we are whitelisting all of deps/ above do we need to do this here?
Also should we be ignoring anything in the deps tree at all? this seems odd to me

Reply:
because the `debug/` pattern re-ignores all paths that have it in anywhere in the path. So we need to do a checkerboard. 
1. ignore `**/debug/**
2. exclude the 3 listed dirs explicitly
3. but ignore build artifacts within them

Probably should try to upstream renaming those directories from `debug` to something like `debugSrc`

RE: the last one I'll double check.